app/controllers/reactivate_account_controller.rb

```diff
@@ -5,10 +5,12 @@ class ReactivateAccountController < ApplicationController
   before_action :confirm_password_reset_profile
 
   def index
+    analytics.reactivate_account_visit
     @personal_key_generated_at = current_user.personal_key_generated_at
   end
 
   def update
+    analytics.reactivate_account_submit
```

Comment:
The thing that's confusing about this to me is that `update` is only called when the user clicks the button "I don't have my key", not when they click the other button that they have their personal key. I think ideally the submit event would capture both of these, but at least for investigative purposes, we do [log the visit on the reactivation screen](https://github.com/18F/identity-idp/blob/d1f0b9320facfad4807643364b2c7ae8662a7617/app/controllers/users/verify_personal_key_controller.rb#L10), which could be enough to see how when a user progresses there.